purism/librem/5r4/kernel/6.1.10.nix

```diff
@@ -0,0 +1,14 @@
+{ buildLinux, fetchFromGitLab, ... }@args:
+buildLinux (args // rec {
+  defconfig = "librem5_defconfig";
+  version = "6.1.10-librem5";
+  modDirVersion = version;
```

Comment:
Does this work with `version = "6.1.10-librem5";`?

Reply:
Yes, it does build and work currently.